electron/gog/games.ts

```diff
@@ -132,8 +132,8 @@ class GOGGame extends Game {
     const etaMatch = data.match(/ETA: (\d\d:\d\d:\d\d)/m)
     const bytesMatch = data.match(/Downloaded: (\S+) MiB/m)
     const progressMatch = data.match(/Progress: (\d+\.\d+) /m)
-    if (etaMatch && bytesMatch && progressMatch) {
-      const eta = etaMatch[1]
+    if (bytesMatch && progressMatch) {
```

Comment:
Removed the eta dependency just in case someone has a pretty slow connection to download a huge game.